type_data/migrations/39_legendary_boxes.rb

```diff
@@ -0,0 +1,4 @@
+migrate :box do
```

Comment:
FYI I renumbered this which will throw off the numbers in your other PR. This is annoying and I think we can come up with a better naming scheme to avoid this issue in the future.